cv/qc-cv.obo

```diff
@@ -417,6 +417,1146 @@ is_a: QC:4000024 ! MS2 metric
+comment: The missed cleavage distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000003 ! single value
+has_units: STATO:0000401 ! sample mean
+synonym: "MS1-5B" NARROW []
+
+[Term]
+id: QC:40000
+name: Missed cleavage sigma
+def: "From the distribution of missed cleavages in identified peptides the sigma value" [PSI:QC]
+comment: The missed cleavage distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000003 ! single value
+has_units: STATO:0000237 ! standard deviation
+
+[Term]
+id: QC:40000
+name: Missed cleavage outliers (<Q1-1.5*IQR, >Q3+1.5*IQR)
+def: "From the distribution of missed cleavages in identified peptides, the list of outliers below of Q1-1.5IQR and above Q3+1.5IQR" [PSI:QC]
+comment: The missed cleavage distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000081 ! Outliers below
+has_column: QC:4000082 ! Outliers above
+
+[Term]
+id: QC:40000
+name: Identified peptide lengths Q1, Q2, Q3
+def: "From the distribution of identified peptide lengths the quartiles Q1, Q2, Q3 value" [PSI:QC]
+comment: The identified peptide lengths distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000004 ! n-tuple
+has_units: STATO:0000167 ! first quartile, STATO:0000574 ! center value, STATO:0000170 ! third quartile
+
+[Term]
+id: QC:40000
+name: Missed cleavage mean
+def: "From the distribution of identified peptide lengths the mean" [PSI:QC]
+comment: The identified peptide lengths distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000003 ! single value
+has_units: STATO:0000401 ! sample mean
+synonym: "MS1-5B" NARROW []
+
+[Term]
+id: QC:40000
+name: Missed cleavage sigma
+def: "From the distribution of identified peptide lengths the sigma value" [PSI:QC]
+comment: The identified peptide lengths distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000009 ! ID based
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000003 ! single value
+has_units: STATO:0000237 ! standard deviation
+
+[Term]
+id: QC:40000
+name: Missed cleavage outliers (<Q1-1.5*IQR, >Q3+1.5*IQR)
+def: "From the distribution of identified peptide lengths, the list of outliers below of Q1-1.5IQR and above Q3+1.5IQR" [PSI:QC]
+comment: The identified peptide lengths distribution can aid the interpretation of digestion efficacy.
+is_a: QC:4000001 ! QC metric
+is_a: QC:4000009 ! ID based
+is_a: QC:4000006 ! table
+has_column: QC:4000081 ! Outliers below
+has_column: QC:4000082 ! Outliers above
+
+[Term]
+id: QC:40000
+name: Signal-to-noise ratio in MS1 Q1, Q2, Q3
```

Comment:
is this computed on raw or profile data? And then one S/N value per data point?
And how is S/N determined?

Reply:
if the algorithm for S/N is deliberately not mentioned, say that the user can select whatever he likes, but the results are not comparable.

Reply:
Same as with FDR for the identification numbers. there is a place for the software in mzQC, which can also detail algorithms used I presume. I'll add a comment for interpretation aid.